dali/kernels/signal/resampling.h

```diff
@@ -54,19 +55,19 @@ inline float32x4_t vsetq_f32(float x0, float x1, float x2, float x3) {
 #endif
 
 struct ResamplingWindow {
-  inline std::pair<int, int> input_range(float x) const {
+  inline DALI_HOST_DEV ivec<2> input_range(float x) const {
```

Comment:
I gather that's because `std::pair` doesn't work with device code? I can't say I'm very happy with (ab)using a vector as start/end.